<launch>
    <arg name="3d_pose" default="ax_pose_estimation_node/3d_pose" />
    <arg name="furniture_json" default="$(find hma_pointing_pkg)/config/json/furniture.json" />
    <arg name="arena_json" default="$(find hma_pointing_pkg)/config/json/arena.json" />
    <arg name="fix_position" default="true" />
    <arg name="start_node" default="false"/>

    <node pkg="hma_pointing_pkg" type="calculate_pointing_position_node_ring_buffer.py" name="calculate_pointing_position_node">
        <param name="/3d_pose" type="str" value="$(arg 3d_pose)"/>
        <param name="/furniture_json" type="str" value="$(arg furniture_json)"/>
        <param name="/arena_json" type="str" value="$(arg arena_json)"/>
        <param name="/fix_position" type="bool" value="$(arg fix_position)"/>
        <param name="/start_node" type="bool" value="$(arg start_node)" />
    </node>

    <node pkg="hma_pointing_pkg" type="env_marker_node.py" name="env_marker_node" >
        <param name="/furniture_json" type="str" value="$(arg furniture_json)"/>
        <param name="/arena_json" type="str" value="$(arg arena_json)"/>
    </node>
</launch>